### ISSUE TITLE
The version of Pytorch_lightning

### ISSUE THREAD

**Issue by yangrongkun** [aminebdj/3D-OWIS#1]
Hello, dear authors, thank you for your great work. 
When I train the code, the version of pytorch_lightning seems to be different, the newest released is 2.0.9,  however, the argument of Trainer.__init__() is not suitable, so what is the detailed version of your pytorch_lightning?

**Comment by aminebdj** [comment]
Hello RongkunYang,

   Thank you for your interest in our work, we used 1.7.2 version of pytorch_lightning. Please find bellow the other dependencies you might need:
 - absl-py==1.3.0
  - addict==2.4.0
  - aiohttp==3.8.3
  - aiosignal==1.3.1
  - albumentations==1.3.0
  - ansi2html==1.8.0
  - antlr4-python3-runtime==4.8
  - appdirs==1.4.4
  - asttokens==2.2.1
  - async-timeout==4.0.2
  - attrs==22.2.0
  - autograd==1.6.2
  - autograd-gamma==0.5.0
  - backcall==0.2.0
  - black==21.4b2
  - cachetools==5.2.0
  - charset-normalizer==2.1.1
  - click==8.1.3
  - cloudpickle==2.1.0
  - comm==0.1.4
  - configargparse==1.7
  - contourpy==1.0.6
  - cycler==0.11.0
  - dash==2.11.1
  - dash-core-components==2.0.0
  - dash-html-components==2.0.0
  - dash-table==5.0.0
  - decorator==5.1.1
  - detectron2==0.6
  - docker-pycreds==0.4.0
  - docutils==0.17.1
  - executing==1.2.0
  - fastjsonschema==2.18.0
  - fire==0.5.0
  - flask==2.2.5
  - fonttools==4.38.0
  - frozenlist==1.3.3
  - fsspec==2022.11.0
  - ftfy==6.1.1
  - future==0.18.3
  - fvcore==0.1.5.post20221221
  - gitdb==4.0.10
  - gitpython==3.1.30
  - google-auth==2.15.0
  - google-auth-oauthlib==0.4.6
  - grpcio==1.51.1
  - hydra-core==1.0.5
  - idna==3.4
  - imageio==2.23.0
  - importlib-metadata==3.10.1
  - iopath==0.1.10
  - ipython==8.14.0
  - ipywidgets==8.1.0
  - itsdangerous==2.1.2
  - jedi==0.19.0
  - jinja2==3.1.2
  - joblib==1.2.0
  - jsonschema==4.18.6
  - jsonschema-specifications==2023.7.1
  - jupyter-core==5.3.1
  - jupyterlab-widgets==3.0.8
  - kiwisolver==1.4.4
  - lightning-utilities==0.5.0
  - loguru==0.6.0
  - markdown==3.4.1
  - markupsafe==2.1.1
  - matplotlib==3.7.1
  - matplotlib-inline==0.1.6
  - minkowskiengine==0.5.4
  - mplcursors==0.5.2
  - multidict==6.0.4
  - mypy-extensions==0.4.3
  - nbformat==5.7.0
  - nest-asyncio==1.5.7
  - networkx==2.8.8
  - ninja==1.10.2.3
  - numpy==1.25.0
  - oauthlib==3.2.2
  - omegaconf==2.0.6
  - open-clip-torch-any-py3==1.3.2
  - open3d==0.17.0
  - opencv-python-headless==4.7.0.68
  - packaging==22.0
  - pandas==2.0.3
  - parso==0.8.3
  - pathspec==0.10.3
  - pathtools==0.1.2
  - pexpect==4.8.0
  - pickleshare==0.7.5
  - pillow==9.4.0
  - platformdirs==3.10.0
  - plotly==5.15.0
  - plyfile==0.7.4
  - pointnet2==0.0.0
  - portalocker==2.6.0
  - promise==2.3
  - prompt-toolkit==3.0.39
  - protobuf==3.20.1
  - psutil==5.9.4
  - ptyprocess==0.7.0
  - pure-eval==0.2.2
  - pyasn1==0.4.8
  - pyasn1-modules==0.2.8
  - pycocotools==2.0.4
  - pydeprecate==0.3.2
  - pygments==2.15.1
  - pyparsing==3.0.9
  - pyquaternion==0.9.9
  - python-dateutil==2.8.2
  - python-dotenv==0.21.0
  - pytorch-lightning==1.7.2
  - pytz==2022.7.1
  - pyviz3d==0.2.32
  - pywavelets==1.4.1
  - pyyaml==5.4.1
  - qudida==0.0.4
  - referencing==0.30.2
  - regex==2022.10.31
  - reliability==0.8.10
  - requests==2.28.1
  - requests-oauthlib==1.3.1
  - retrying==1.3.4
  - rpds-py==0.9.2
  - rsa==4.9
  - scikit-image==0.19.3
  - scikit-learn==1.2.0
  - scipy==1.11.1
  - seaborn==0.12.2
  - sentry-sdk==1.12.1
  - setproctitle==1.3.2
  - shortuuid==1.0.11
  - six==1.16.0
  - smmap==5.0.0
  - stack-data==0.6.2
  - tabulate==0.9.0
  - tenacity==8.2.2
  - tensorboard==2.11.0
  - tensorboard-data-server==0.6.1
  - tensorboard-plugin-wit==1.8.1
  - tensorboardx==2.5.1
  - termcolor==2.1.1
  - threadpoolctl==3.1.0
  - tifffile==2022.10.10
  - toml==0.10.2
  - torch==1.12.1+cu116
  - torch-scatter==2.1.0+pt112cu116
  - torchmetrics==0.11.0
  - torchvision==0.13.1+cu116
  - tqdm==4.64.1
  - traitlets==5.9.0
  - trimesh==3.17.1
  - typing-extensions==4.4.0
  - tzdata==2023.3
  - urllib3==1.26.13
  - volumentations==0.1.8
  - wandb==0.13.7
  - wcwidth==0.2.6
  - werkzeug==2.2.3
  - widgetsnbextension==4.0.8
  - yacs==0.1.8
  - yarl==1.8.2
  - zipp==3.11.0
